pkg/model/api/v1/utils/default.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import v1 "github.com/perses/perses/pkg/model/api/v1"
+
+var (
+	owner  = "owner"
+	editor = "editor"
+	viewer = "viewer"
+)
+
+func DefaultOwnerRole(projectName string) v1.Role {
```

Comment:
if you agree with having a package role, then this data could go there and be defined like `role.DefaultOwner`

Reply:
I can't put this file in role => it importing v1 and v1 import role 😞